components/binance/browser/regions.h

```diff
@@ -33,6 +33,11 @@ const std::vector<std::string> unsupported_regions = {
   "ZW"
 };
 
+const std::vector<std::string> supported_locales = {
+    "en", "au", "cn", "tw", "ar", "nl", "ph", "fr", "de", "id",
+    "it", "ja", "kr", "pl", "br", "pt", "ru", "es", "th", "tr",
+    "ua", "vn", "ro", "cs", "he", "bg", "lv", "bn", "sv"};
```

Comment:
Does this match the locale list that Binance provided?

Reply:
Yep, double checked, these are all the locales supported by binance.com